devlib/platform/arm.py

```diff
@@ -13,9 +13,9 @@
 # limitations under the License.
```

Comment:
This was because in the first version the Gem5SimulationPlatform was placed in this file